acceptancetests/run-windows-job.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#This script allows you to run arbitrary commands on windows slaves
+#You can also specify a tarball to be used as part of the payload
+#Usage: host (tarball or revision id or cmd_to_run)
+set -eux
+IFS='%'
+
+runcmd() {
+    #Make payload to run
+    payload="$ts.bat"
+    echo "cd C:\\Users\\Administrator\\payloads\\$ts" > $payload
+    echo "$1" >> $payload
+    #Ensure the error is returned if the command fails
+    echo "if errorlevel 1 (" >> $payload
+    echo "   exit /b %errorlevel%" >> $payload
+    echo ")" >> $payload
+    scp -i $JUJU_HOME/staging-juju-rsa -oStrictHostKeyChecking=no $payload $host:C:\\Users\\Administrator\\payloads\\
+    #Run test command via ssh
+    ssh $host C:\\Users\\Administrator\\payloads\\$payload
+}
+
+host=$1
+ts=$(date +%s)
+tarfile=''
+echo "Running on $host as payload $ts"
+runcmd "mkdir C:\\Users\\Administrator\\payloads\\$ts"
```

Comment:
I would prefer these variables to be setup at the to before the function, purely to save having to bounce up and down to see what $ts is within the function.

Also can we have host=$1 for completeness :-)